app/Main.hs

```diff
@@ -424,38 +423,43 @@ checkForUpdates applyMinorUpdates applyMajorUpdates = do
     isMinorReleaseFrom (x : xs) (y : ys) = y == x && ys > xs
     isMinorReleaseFrom _        _        = False
 
-verify :: String -> IO ()
-verify inputName = case mkPackageName (pack inputName) of
+data VerifyArgs a = Package a | After a | NoArg
```

Comment:
You can derive `Traversable` here and use that to move the `mkPackageName` call up. 

Also, I feel like `After` and `NoArg` should be combined into a single `VerifyAll` constructor, and use a `Maybe` for the package name.

Reply:
How does the usage text from `psc-package verify --help` look now?

Reply:
To be clear, the reason I prefer one `VerifyAll` constructor is that I think the constructors here should mirror the subcommands, and flags should be represented as constructor arguments. But also, I think it might improve the usage text and help to reduce a little code duplication.

Reply:
The usage text:
```
Usage: psc-package verify ([PACKAGE] | [--after ARG])
  Verify that the named package builds correctly. If no package is specified,
  verify that all packages in the package set build correctly.

Available options:
  PACKAGE                  The name of the package to install
  --after ARG              Skip packages before this package during verification
  -h,--help                Show this help text
```